common/config/babel/.babelrc.js

```diff
@@ -113,7 +113,11 @@ process.env['COMMUNICATION_REACT_FLAVOR'] !== 'beta' &&
         // Optimal Video Count
         'optimal-video-count',
         // raise hands feature
-        'raise-hand'
+        'raise-hand',
+        // Feature for gallery layouts
+        'gallery-layouts',
+        // feature  for vertical overflow gallery
+        'vertical-gallery'
```

Comment:
are we stabilizing `vertical-gallery` as well?

Reply:
yup both features are going in this release as part of gallery layouts